src/models/endpoint/facade.js

```diff
@@ -0,0 +1,55 @@
+/*****
```

Comment:
Please align the `model` directory `endpoint` to `participant` as per the Central-Ledger.